client/android/app/build.gradle

```diff
@@ -84,7 +84,7 @@ def enableProguardInReleaseBuilds = false
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    buildToolsVersion '25.0.0'
```

Comment:
When I open the project it displays warnings saying that you aren't using the latest version of the build tools and this may cause problems.